src/components/strings.jl

```diff
@@ -149,23 +153,43 @@ function parse_string_or_cmd(ps::ParseState, prefixed=false)
                         pushtotrivia!(ret, rparen)
                         skip(input, 1)
                     else
-                        interp_val = @closer ps1 :paren parse_expression(ps1)
+                        interp_val = @closer ps1 :paren parse_expression(ps1, true)
                         push!(ret, interp_val)
                         pushtotrivia!(ret, op)
                         pushtotrivia!(ret, lparen)
-                        pushtotrivia!(ret, rparen)
-                        seek(input, ps1.nt.startbyte + 1)
+                        if kindof(ps1.nt) === Tokens.RPAREN
+                            # Need to check the parenthese were actually closed.
+                            pushtotrivia!(ret, rparen)
+                            seek(input, ps1.nt.startbyte + 1)
+                        else
+                            pushtotrivia!(ret, EXPR(:RPAREN, 0, 0))
+                            if eof(input)
+                                # Some errored interpolation, we're now at the end of the string but lets make sure we have the closing "
+                                # write(b, '"')
+                                # erroredonlast = true
+                                # @info "break"
+                            #    break
```

Comment:
Is this meant to be carried forward, or could this be deleted? If it should stay, maybe add a bit more explanation what it is for some future person trying to understand this :)

Reply:
Nope just some leftover junk!